actions/tag/delete.go

```diff
@@ -0,0 +1,46 @@
+package tag
+
+import (
+	"encoding/json"
+	"github.com/factly/data-portal-api/models"
+	"github.com/factly/data-portal-api/validation"
+	"github.com/go-chi/chi"
+	"net/http"
+	"strconv"
+)
+
+
+// DeleteTag - Delete tag by id
+// @Summary Delete a tag
+// @Description Delete tag by ID
+// @Tags Tag
+// @ID delete-tag-by-id
+// @Consume  json
+// @Param id path string true "Tag ID"
+// @Success 200 {object} models.Tag
+// @Failure 400 {array} string
+// @Router /tags/{id} [delete]
+func deleteTag(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think we should change the function name also to `delete`. it's function inside tag package so it won’t affect other part also.